wiki/osu!stream/de.md

```diff
@@ -1,19 +1,22 @@
+
+Im Gegensatz zum normalen osu! enthält osu!stream ein komplett überarbeitetes Spielsystem mit neuen Spielelementen wie den Hold-Circles und verbundenen Beats. Klassische osu! Beatmaps werden nicht ohne manuelle Umwandlung unterstützt. Stattdessen wurden neue Beatmaps explizit von einem besonderen Team an Mappern erstellt, um die Chance der neuen Spielelemente auszunutzen. Diese neuen Beatmaps haben einen [strikteren Standard](https://docs.google.com/document/d/1FYmHhRX-onR-osgTS6uHSOZuu_0JEbfRZePVySvvr9g).
 
-Enthalten ist ein komplett überarbeitetes Spielsystem mit neuen Spielelementen wie den Hold-Circles und verbundenen Beats. Klassische osu! Beatmaps werden nicht ohne manuelle Umwandlung unterstützt. Stattdessen wurden neue Beatmaps explizit von einem besonderen Team an Mappern erstellt, um die Chance der neuen Spielelemente auszunutzen. Diese neuen Beatmaps haben einen [strikteren Standard](https://docs.google.com/document/d/1FYmHhRX-onR-osgTS6uHSOZuu_0JEbfRZePVySvvr9g).
+## Geschichte
 
-[osu!stream wurde in osu!monthly \#5 zusammen mit einem Interview vorgestellt.](https://osu.ppy.sh/community/forums/topics/59924)
+Die Entwicklung von osu!stream begann im Jahr 2011 als ein Versuch, eine Version von osu! zu veröffentlichen, die unterwegs gespielt werden konnte, in Anlehnung an das zuvor aufgegebene Projekt [osu! iPhone](https://github.com/ppy/osu-iPhone). Zu Anfang war es als eine iOS-exklusive App gedacht, da die Infrastruktur von Android zur damaligen Zeit bedeutete, dass Audio-Latenzprobleme auf der Plattform unvermeidlich waren.
 
-Die Entwicklung für diese Version endete in 2020. Für mehr Informationen lies den [Blog-Beitrag](https://blog.ppy.sh/osu-stream-2020-release/) über ihre letzte Veröffentlichung. Der Quellcode zusammen mit den Beatmap-Konvertierungs- und Test-Tools wurde im selben Jahr unter einer freien Lizenz auf [GitHub](https://github.com/ppy/osu-stream) zur Verfügung gestellt. Alle zuvor kostenpflichtigen Beatmaps sind jetzt kostenlos im in-game "Store" erhältlich.
+Das Spiel wurde erstmals in einem [kryptischen YouTube-Video](https://www.youtube.com/watch?v=6Jyzgm_-joM) im Juni 2011 angeteasert. Nach der Veröffentlichung ein paar Monate später wurden die Hintergründe des Spiels in [osu!monthly \#5](https://osu.ppy.sh/community/forums/topics/59924) zusammen mit einem Interview mit peppy erläutert.
+
+Die Hauptentwicklung für osu!stream wurde 2020 eingestellt. Lies für mehr Informationen den [Blog-Beitrag](https://blog.ppy.sh/osu-stream-2020-release/) zu seiner letzten Veröffentlichung. Der Quellcode zusammen mit den Beatmap-Konvertierungs- und Test-Tools wurde im selben Jahr unter einer freien Lizenz auf [GitHub](https://github.com/ppy/osu-stream) zur Verfügung gestellt. Alle zuvor kostenpflichtigen Beatmaps sind jetzt kostenlos im "In-Game-Store" erhältlich.
```

Comment:
```suggestion
Die Hauptentwicklung für osu!stream wurde 2020 eingestellt. Für weitere Informationen lies [Blog-Beitrag](https://blog.ppy.sh/osu-stream-2020-release/) bezüglich der letzten Veröffentlichung. Der Quellcode zusammen mit den Beatmap-Konvertierungs- und Test-Tools wurde im selben Jahr unter einer freien Lizenz auf [GitHub](https://github.com/ppy/osu-stream) zur Verfügung gestellt. Alle zuvor kostenpflichtigen Beatmaps sind jetzt kostenlos im "In-Game-Store" erhältlich.
```

Reply:
Do you think the original text sounds better? After reading through the text while adding in the updates, I thought that this phrase sounds clunky. Thus, I changed it.

An alternative I could imagine as well would be `Weitere Informationen findest du im Blogbeitrag zur letzten Veröffentlichung.`

Reply:
Yeah, I feel like `Lies für mehr Informationen …` reads less natural, but not sure. I like your other alternative though, so let's go with that. I edited my suggestion. Also getting rid of the unnecessary hyphen. :+1: